backend/src/Designer/Configuration/ResourceRegistryBaseUrlSetting.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Altinn.Studio.Designer.Configuration
+{
+    public class ResourceRegistryBaseUrlSetting
```

Comment:
I dont remember exactly, but I was expecting some info about which Maskinporten environment to use per resource registry settings. Did we decide to add it later?